packages/extension-chakra-storefront/package.json

```diff
@@ -79,13 +79,16 @@
     "@types/react": "~18.2.0",
     "@types/react-dom": "~18.2.1",
     "express": "^4.19.2",
-    "internal-lib-build": "4.0.0-dev",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "engines": {
     "node": "^16.11.0 || ^18.0.0 || ^20.0.0",
     "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
+  },
+  "mobify": {
+    "app": {
+      "extensions": []
+    }
```

Comment:
Right now the babel config from pwa-kit-dev has extensibility support baked into it. Which means it's looking for an application configuration file all the time. This is a work around for now, but I've created [this](https://gus.lightning.force.com/a07EE000026NFQlYAO) ticket to fix it.